app/assets/stylesheets/pages/_subscription.scss

```diff
@@ -0,0 +1,10 @@
+.notice-box {
```

Comment:
@Tallys This should probably be in global instead of a page, since it's used for more than one page.